libnest/src/package/requirement.rs

```diff
@@ -38,16 +38,16 @@ impl PackageRequirement {
 
     /// Creates a package requirement that matches the given [`PackageFullName`] and version requirement.
     #[inline]
-    pub fn from_id(id: &PackageID) -> PackageRequirement {
-        PackageRequirement {
+    pub fn from_id(id: &PackageID) -> SoftPackageRequirement {
```

Comment:
This somehow wasn't implemented as a `From<PackageID>`, I opened an issue for it: see #100